tests/integration.js

```diff
@@ -51,31 +53,19 @@ describe( 'Integration of CKEditor component', () => {
 		} );
 	} );
 
-	it.skip( 'should use correct CKEDITOR build', () => {
-		const basePath = 'https://cdn.ckeditor.com/4.13.0/standard-all/';
-
-		delete window.CKEDITOR;
-
-		return createComponent( { editorUrl: basePath + 'ckeditor.js' } ).then( () => {
-			expect( window.CKEDITOR.basePath ).to.equal( basePath );
-		} );
-	} );
-
-	it.skip( 'should call namespace loaded directive only for the initial script load', () => {
+	it( 'should call namespace loaded directive only for the initial script load', () => {
 		const spy = sinon.spy();
 
-		delete window.CKEDITOR;
-
 		return Promise.all( [
 			createComponent( {}, spy ),
 			createComponent( {}, spy ),
 			createComponent( {}, spy )
 		] ).then( () => {
-			expect( spy.calledOnce ).to.equal( true );
+			expect( spy.callCount ).to.equal( 1 );
```

Comment:
As we use `sinon` + `chai` we could write nicer assertions using [`sinon-chai`](https://www.npmjs.com/package/sinon-chai).

Reply:
Could you tell what will be nicer from your perspective? :> This looks good for me. It's short, and tells you exactly the difference between calls count.

Reply:
I'm ok with both versions, but the best one for me is:

```javascript
expect( spy ).to.have.been.calledOnce;
```

:wink:

Reply:
If you ok with both versions - I would keep `callCount`. Not sure if I mentioned that somewhere, but it is a huge difference (eg. in source of the bug) if the callback wasn't called at all or was called to many times. Current assertion gives us an exact number of calls in case of fail.